FsSonarRunner/FsSonarRunner/Program.fs

```diff
@@ -46,8 +46,14 @@ let main argv =
                 let options = XmlHelper.InputXml.Parse(File.ReadAllText(input))
 
                 let metrics = new SQAnalyser()

```

Comment:
`SQAnalyser` is not an implementation of `IDisposable`, is it? Then the keyword `new` can be omitted.

Reply:
sure, c# habits